Demo/Measures-cms/AHAOverall-2.6.000.g.cs

```diff
@@ -285,28 +285,32 @@ private Patient Patient_Value()
 		};
 		bool? b_(Condition Diagnosis)
 		{
-			var f_ = FHIRHelpers_4_3_000.ToConcept(Diagnosis?.ClinicalStatus);
-			var g_ = QICoreCommon_2_0_000.active();
-			var h_ = context.Operators.ConvertCodeToConcept(g_);
-			var i_ = context.Operators.Equivalent(f_, h_);
-			var j_ = FHIRHelpers_4_3_000.ToConcept(Diagnosis?.VerificationStatus);
-			var k_ = QICoreCommon_2_0_000.unconfirmed();
-			var l_ = context.Operators.ConvertCodeToConcept(k_);
-			var m_ = context.Operators.Equivalent(j_, l_);
-			var o_ = QICoreCommon_2_0_000.refuted();
-			var p_ = context.Operators.ConvertCodeToConcept(o_);
-			var q_ = context.Operators.Equivalent(j_, p_);
-			var r_ = context.Operators.Or(m_, q_);
-			var t_ = QICoreCommon_2_0_000.entered_in_error();
-			var u_ = context.Operators.ConvertCodeToConcept(t_);
-			var v_ = context.Operators.Equivalent(j_, u_);
-			var w_ = context.Operators.Or(r_, v_);
-			var x_ = context.Operators.Not(w_);
-			var y_ = context.Operators.And(i_, x_);
-
-			return y_;
+			var f_ = Diagnosis?.ClinicalStatus;
+			var g_ = FHIRHelpers_4_3_000.ToConcept(f_);
+			var h_ = QICoreCommon_2_0_000.active();
+			var i_ = context.Operators.ConvertCodeToConcept(h_);
+			var j_ = context.Operators.Equivalent(g_, i_);
+			var k_ = Diagnosis?.VerificationStatus;
+			var l_ = FHIRHelpers_4_3_000.ToConcept(k_);
+			var m_ = QICoreCommon_2_0_000.unconfirmed();
+			var n_ = context.Operators.ConvertCodeToConcept(m_);
+			var o_ = context.Operators.Equivalent(l_, n_);
+			var q_ = FHIRHelpers_4_3_000.ToConcept(k_);
+			var r_ = QICoreCommon_2_0_000.refuted();
+			var s_ = context.Operators.ConvertCodeToConcept(r_);
+			var t_ = context.Operators.Equivalent(q_, s_);
+			var u_ = context.Operators.Or(o_, t_);
+			var w_ = FHIRHelpers_4_3_000.ToConcept(k_);
+			var x_ = QICoreCommon_2_0_000.entered_in_error();
+			var y_ = context.Operators.ConvertCodeToConcept(x_);
+			var z_ = context.Operators.Equivalent(w_, y_);
+			var aa_ = context.Operators.Or(u_, z_);
+			var ab_ = context.Operators.Not(aa_);
+			var ac_ = context.Operators.And(j_, ab_);
+
+			return ac_;
 		};
-		var c_ = context.Operators.Where<Condition>(a_, b_);
+		var c_ = context.Operators.Where<Condition>((IEnumerable<Condition>)a_, b_);
```

Comment:
Is this cast expected?

Reply:
It isn't required, but there is still breaking changes happening elsewhere if this cast is removed globally